### DISCUSSION TITLE
Can milvus use the indices files generated by faiss?

### DISCUSSION THREAD

**Discussion by prprprus** [milvus-io/milvus#4219] [Q&A and General discussion] [answered]
1. Can milvus use the indices files generated by faiss?
2. Is the milvus production environment ready?

**Comment by shiyu22** [comment, marked as answer]
1. Milvus can't use Faiss indices files, you can use Milvus itself to [insert the vector](https://milvus.io/docs/v0.11.0/insert_delete_entity.md) and [create the index](https://milvus.io/docs/v0.11.0/create_drop_index.md).
2. Milvus is definitely production-ready, and there are many users who put it into production.

**Reply by prprprus** [reply]
Thanks
